src/styles/layout/_component-grid.scss

```diff
@@ -122,13 +118,14 @@ $breakpoint-accordion-medium-550: 34rem;
     &--max-scott {
       grid-area: max-scott;
       background-color: transparent;
-      max-width: 17.1875rem; // 275px/16
       height: auto;
       margin: auto;
+      max-width: 17.1875rem;
 
       img {
-        object-position: inherit;
-        object-fit: inherit;
+        width: 100%;
+        height: 100%;
+        object-fit: contain;
       }
```

Comment:
The Max on the homepage should scale with screen size now.

Reply:
This ended up being trickier than I thought. In 58d283f, I added a container around the image of max so that max+the doodles don't bleed over to the left and right as much:
![Screen Shot 2022-12-06 at 2 16 09 PM](https://user-images.githubusercontent.com/55593075/206001836-3adb7f53-0e34-4747-a5d1-b06e553c39c8.png)
This commit also makes sure max doesn't get any of the doodles from the nth-child() rules.